content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -649,32 +666,30 @@ cap on the amount of resources that a particular namespace can
 consume.
 {{< /note >}}
 
-
 ## Job patterns
 
-The Job object can be used to support reliable parallel execution of Pods.  The Job object is not
+The Job object can be used to support reliable parallel execution of Pods. The Job object is not
 designed to support closely-communicating parallel processes, as commonly found in scientific
```

Comment:
NB: This is not true, but I'll deal with it in a separate PR.

Reply:
Thanks